### PR TITLE
Return level to peatio

### PR DESCRIPTION
**Pull request by vpetrusenko**

### CODE REVIEW

File: app/models/account.rb [hunk 1]
```diff
@@ -56,6 +57,11 @@ def level_set(step)
 
     save
   end
+
+  def hook_req
+    req_params = to_json(only: %i[email level])
+    RestClient.put(ENV['peatio_webhook'].to_s, req_params)
```
Comment (yssavchuk) [inline app/models/account.rb]: I think it is better to use ENV.fetch['PEATIO_WEBHOOK_URL']

Comment (vpetrusenko) [reply app/models/account.rb]: Rewrite according to this review

File: app/models/account.rb [hunk 2]
```diff
@@ -56,6 +57,11 @@ def level_set(step)
 
     save
   end
+
+  def hook_req
```
Comment (gfedorenko) [inline app/models/account.rb]: Should try to put it into level_set

Comment (vpetrusenko) [reply app/models/account.rb]: @gfedorenko rewrite according to this changes

**Comment by mod** [comment]
There is many things wrong about this patch,
peatio can access barong API you guys have to learn about OAuth 2.0 protocol.
https://www.youtube.com/results?search_query=oauth+2.0+protocol+explained

**Pull request closed without merging**